### DISCUSSION TITLE
Anyone think of VLC Integration?

### DISCUSSION THREAD

**Discussion by JaminShanti** [jamulussoftware/jamulus#2072] [Ideas]
Anyone think of doing a VLC integration from webpage listing of rooms?  So you can listen while your on the road, even if you don't have time to jam.  Or for audiences to tune in.  They have VLC for All major Platforms and it supports streaming.
Saw someone post about it on Source forge here, https://sourceforge.net/p/llcon/discussion/533517/thread/6d4e212df1/
I wondered how hard it would be to integrate this directly into Jamulus, so every room supported it, public or private.

**Comment by WolfganP** [comment]
Search for "jamulus icecast", there are several tutorials of successful integrations to cast a jamulus mix as an internet radio.

**Comment by gilgongo** [comment]
See also this PR for  `--streamto` to make ffmpeg stream to an Icecast server: https://github.com/jamulussoftware/jamulus/pull/967 Not sure what the status of it is though.  EDIT: Also this implementation: https://github.com/dtinth/jamulus/commit/f534e8cbe0688069ce50b3613cfd407fffa39642
